drivers/pulseaudio/audio_driver_pulseaudio.cpp

```diff
@@ -225,51 +227,96 @@ Error AudioDriverPulseAudio::init_device() {
 	return OK;
 }
 
+#define LIBPULSE_DLSYM(name)                                                 \
```

Comment:
Might be worth moving all this dlopening work to another file IMO and include it here.